packages/apps/esm-primary-navigation-app/src/index.ts

```diff
@@ -53,3 +54,8 @@ export const changeLanguageModal = getAsyncLifecycle(
   () => import('./components/change-language/change-language.modal'),
   options,
 );
+
+export const globalNavGroup = getSyncLifecycle(GlobalNavGroup, {
```

Comment:
Use `getAsyncLifecycle()` here please.

```suggestion
export const globalNavGroup = getSyncLifecycle(GlobalNavGroup, {
```